sqldelight-gradle-plugin/src/test/fixtures/well-formed-selects/build.gradle

```diff
@@ -0,0 +1,13 @@
+plugins {
+  id 'com.android.application'
+  id 'com.squareup.sqldelight'
+}
+
+android {
+  compileSdkVersion 23
+  buildToolsVersion '23.0.2'
+
+  sourceSets {
+    main.java.srcDirs += '../../../../../sqldelight-runtime/src/main/java'
```

Comment:
This should probably be part of the default `build.gradle`. It won't hurt to have it in the failure cases and in the passing cases it means you can omit this file and let the default take over.